src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,77 @@
+import timeSince from "../../utils/timeSince";
+import styles from "./Comment.module.scss";
+import blankProfile from "../../image/blankProfile.svg";
+import { useState } from "react";
+export default function Comment({ comment, handleModify, handleDelete }) {
+  const { content, updatedAt, id, writer } = comment;
+  const [show, setShow] = useState(false);
```

Comment:
💬 여담
reducer 를 사용해 아래처럼 간단하게도 작성 가능합니다~

```suggestion
const [isShow, toggleIsShow] = useReducer((prev) => !prev, false);
```